### PR TITLE
Add AW and WWP issues ID

### PR DESCRIPTION
**Pull request by antoscarface**
Give me the ability to use these fantastic monkey tools also with the website and plugin repositories 💃 

Tested and seems work like a charm!

### CODE REVIEW

File: github/jira-issue.user.js [hunk 1]
```diff
@@ -5,15 +5,15 @@
 // @description  Linkifies branch name
 // @author       Massimiliano Cannarozzo
 // @updateURL    https://raw.githubusercontent.com/adespresso/monkey-tools/master/github/jira-issue.user.js
-// @include      /^https://github.com/adespresso/adespresso/pull/\d+/
+// @include      /^https://github.com/adespresso/(adespresso|pixel-caffeine\website\university)/pull/\d+/
```
Comment (maxcanna) [inline github/jira-issue.user.js]: Maybe we could make it more generic

File: github/jira-sidebar-item.user.js [hunk 2]
```diff
@@ -7,7 +7,7 @@
 // @updateURL    https://raw.githubusercontent.com/adespresso/monkey-tools/master/github/jira-sidebar-item.user.js
 // @grant        GM_getResourceText
 // @grant        GM_xmlhttpRequest
-// @include      /^https://github.com/adespresso/adespresso/pull/\d+/
+// @include      /^https://github.com/adespresso/(adespresso|pixel-caffeine\website\university)/pull/\d+/
```
Comment (maxcanna) [inline github/jira-sidebar-item.user.js]: Moaaaar generic please :)

File: github/jira-sidebar-item.user.js [hunk 3]
```diff
@@ -146,5 +146,6 @@ function main() {
 
 (function () {
     waitForKeyElements(".head-ref:contains('AAPP'),h1:contains('AAPP')", main);
+    waitForKeyElements(".head-ref:contains('AW'),h1:contains('AW')", main);
+    waitForKeyElements(".head-ref:contains('WWP'),h1:contains('WWP')", main);
```
Comment (maxcanna) [inline github/jira-sidebar-item.user.js]: 🚨 Please avoid calling the function multiple times

**Comment by antoscarface** [comment]
@maxcanna Requested changes are done!

**Comment by maxcanna** [comment]
@antoscarface please update readme and changelog

**Comment by antoscarface** [comment]
@maxcanna updated changelog, I rebased in order to grab the last changes merged some minutes ago in the changelog. 
Instead, I don't have anything to update in the README.md.

**Comment by antoscarface** [comment]
Thank you @maxcanna

**Pull request merged**